docs/Quickstart.md

```diff
@@ -10,13 +33,13 @@ There's a lot of scenarios that can be encountered when you want to create a moc
 
 ### API Generation
 
-With Rocks 5.0.0 and above, all of the mocks are created using [C# 9.0's source generation feature](https://devblogs.microsoft.com/dotnet/introducing-c-source-generators/). This means that you must be targeting .NET 5.0 to use Rocks. With source generation, you have a lot of freedom to generate what you want, and this is exactly what Rocks takes advantage of. When you want to create a mock, a number of extension methods are generated that group the members you can set expectations on. You'll see in this document references in code to `.Methods()`, `.Properties()`, and `.Indexers()`. You use these extension methods to set expectations on specific members. This should become clear as you read on and look at the examples.
+With Rocks 5.0.0 and above, all of the mocks are created using [C# 9.0's source generation feature](https://devblogs.microsoft.com/dotnet/introducing-c-source-generators/ "Introducing C# Source Generators - Microsoft"). This means that you must be targeting .NET 5.0 to use Rocks. With source generation, you have a lot of freedom to generate what you want, and this is exactly what Rocks takes advantage of. When you want to create a mock, a number of extension methods are generated that group the members you can set expectations on. You'll see in this document references in code to `.Methods()`, `.Properties()`, and `.Indexers()`. You use these extension methods to set expectations on specific members. This should become clear as you read on and look at the examples.
```

Comment:
Not sure why the source generator link here has to change.

Reply:
The quote just adds the title of the page as an alt text if you hover over the link. Similar alt texts have been added to the other links as a small qol improvement. The actual links have no change, they still point to the original urls.